tests/regression_tests/test_llama2_7b.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import re
+import runpy
+import sys
+from pathlib import Path
+
+import pytest
+import torchtune
+from tests.common import TUNE_PATH
+from tests.test_utils import CKPT_MODEL_PATHS, gpu_test
+
+
+CKPT = "llama2_7b"
+
+# TODO: remove this once we have eval configs exposed properly
+pkg_path = Path(torchtune.__file__).parent.absolute()
+EVAL_CONFIG_PATH = Path.joinpath(
+    pkg_path, "_cli", "eval_configs", "default_eval_config.yaml"
+)
+
+
+@gpu_test(gpu_count=2)
+class TestLoRA7BDistributedFinetuneEval:
```

Comment:
Nice! Any reason we do this only for LoRA?

Reply:
Ugh this is the one piece I have not ironed out yet. I had the full finetune test_loss in here on 7B but [took it out](https://github.com/pytorch/torchtune/pull/537/commits/b60e9d41eab0062a35e44a791ea8818453454227) just for testing. Basically all our regression tests are OOMing on the CI hardware, so I am trying to sort that out. Once it's figured out I will add the full finetune loss test back (but probably not an eval test, given that the eval results on many full finetunes are questionable anyways)